src/slack_block_util.ts

```diff
@@ -136,6 +136,17 @@ const volunteerSelectionPanel: SlackBlock = {
         emoji: true,
       },
     },
+    {
+      type: 'button',
+      style: 'primary',
+      text: {
+        type: 'plain_text',
+        text: 'Release claim',
```

Comment:
I wonder if "Clear volunteer" is clearer for the button text

Reply:
updated!